### PR TITLE
Add joint velocity limits task and switch to `toml` configuration files in tests

### PR DESCRIPTION
**Pull request by davidegorbani**
This PR adds a task to add joint velocity limits and switches the type of configuration files in tests from `ini` to `toml`.

### CODE REVIEW

**Review by traversaro — APPROVED** [review]

File: src/IK/src/InverseKinematics.cpp [hunk 1]
```diff
@@ -359,6 +366,7 @@ bool HumanIK::advance()
 {
     // Initialize ok flag to true
     bool ok{true};
+    // m_jointVelocityLimitsTask->update();
```
Comment (Gianlucamilani) [inline src/IK/src/InverseKinematics.cpp:369]: is this comment useful?

Comment (davidegorbani) [reply src/IK/src/InverseKinematics.cpp:369]: It is a leftover, thank you.

Comment (davidegorbani) [reply src/IK/src/InverseKinematics.cpp]: removed with https://github.com/ami-iit/biomechanical-analysis-framework/pull/43/commits/b28b7e4667643905fbaea19371517e83e3fd9d60

**Comment by davidegorbani** [comment]
The CI is failing because the task `JointVelocityLimitsTask` is not available in the latest version of `blf` on conda; I would merge this PR and then fix the CI.

**Comment by traversaro** [comment]
> The CI is failing because the task `JointVelocityLimitsTask` is not available in the latest version of `blf` on conda; I would merge this PR and then fix the CI.

This will be fixed by https://github.com/conda-forge/bipedal-locomotion-framework-feedstock/pull/81 in 2/3 hours.

**Comment by davidegorbani** [comment]
> > The CI is failing because the task `JointVelocityLimitsTask` is not available in the latest version of `blf` on conda; I would merge this PR and then fix the CI.
> 
> This will be fixed by [conda-forge/bipedal-locomotion-framework-feedstock#81](https://github.com/conda-forge/bipedal-locomotion-framework-feedstock/pull/81) in 2/3 hours.

Great, thank you! Then I would wait for it!

**Comment by traversaro** [comment]
Ok, I was a bit optimistic, the packages were uploaded ~1 hour ago: https://anaconda.org/conda-forge/bipedal-locomotion-framework/files, hopefully they should be CDN propagated now or in a few minutes.

**Comment by davidegorbani** [comment]
Merging as the CI was successful!

**Pull request merged**